spec.html

```diff
@@ -30581,7 +30587,7 @@ <h1>Symbol.prototype [ @@toPrimitive ] ( _hint_ )</h1>
 
       <emu-clause id="sec-symbol.prototype-@@tostringtag">
         <h1>Symbol.prototype [ @@toStringTag ]</h1>
-        <p>The initial value of the @@toStringTag property is the String value *"Symbol"*.</p>
+        <p>The initial value of the @@toStringTag property is *"Symbol"*.</p>
```

Comment:
Not to self: put this back.

```suggestion
        <p>The initial value of the @@toStringTag property is the String value *"Symbol"*.</p>
```